core/src/main/kotlin/no/nav/amt/tiltak/core/domain/tiltak/ArrangorVeiledersDeltaker.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.amt.tiltak.core.domain.tiltak
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class ArrangorVeiledersDeltaker(
```

Comment:
Jeg skjønner det må være her for den store queryen, men dette er vel egentlig flere domeneobjekter, ikke ett - i mine øyne.

Reply:
Det er den informasjonen frontend trenger å få fra apiet - derav et objekt.

Reply:
Jeg har forsåvidt ikke noe ønske om å definere dette objektet her i core - jeg er enig i at det ikke hører hjemme der siden det kun er noe tiltaksarrangør-bff-en trenger. Men siden koden dessverre er delt inn på denne måten med disse interfacene som ligger i core som alle må implementere har jeg ikke noe valg :(

Reply:
Da blir koden ekstremt ineffektiv og rotete :(